css/cssom-view/elementsFromPoint-display-inline.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSSOM View - 5 - extensions to the Document interface</title>
+    <link rel="author" title="Luke Warlow" href="mailto:lwarlow@igalia.com">
+    <link rel="help" href="http://www.w3.org/TR/cssom-view/#extensions-to-the-document-interface">
+    <meta name="flags" content="dom">
```

Comment:
I think at least `css/cssom-view/elementsFromPoint-inline-*.html` cover this already...